### DISCUSSION TITLE
How to get multiple layout styles in the same workspace?

### DISCUSSION THREAD

**Discussion by leminhman0312** [i3/i3#5671] [Q&A]
Basically this, https://i3wm.org/docs/modes.png, how to get the left 2 windows as tiles,  the middle 2 windows in stacking mode and the last 2 windows in tabbed mode? 

If I start with 2 windows and set them in tabbed mode, then the 3rd window that I open would be stuck in the tabbed mode.  How to set this 3rd window as tiling while keeping the 1st and 2nd in tabbed mode?

**Comment by okraits** [comment]
If you turn an individual window into a split container, you can then change the layout of that split container:

https://i3wm.org/docs/userguide.html#_splitting_containers

This is also explained in the screencast videos: https://i3wm.org/screenshots/
